src/main/java/mate/academy/dao/CinemaHallDao.java

```diff
@@ -10,4 +10,8 @@
     Optional<CinemaHall> get(Long id);
 
     List<CinemaHall> getAll();
+
+    boolean update(CinemaHall cinemaHall);
+
+    boolean delete(Long id);
```

Comment:
Consider changing the return type of the `delete` method to `void` or throwing an exception if the deletion fails. Returning a boolean might not be the best approach for error handling.